elem.go

```diff
@@ -62,6 +62,16 @@ func (t TextNode) Render() string {
 	return string(t)
 }
 
+type CommentNode string
+
+func (c CommentNode) RenderTo(builder *strings.Builder) {
```

Comment:
Great implementation of the `CommentNode`! One small suggestion for optimization:

```go
func (c CommentNode) RenderTo(builder *strings.Builder) {
	builder.WriteString("<!-- ")
	builder.WriteString(string(c))
	builder.WriteString(" -->")
}
```

Reply:
Thanks for this performance tip. Fixed.